### DISCUSSION TITLE
Suborbital should perhaps be safer

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#584] [Ideas]
The heat of reentry isn't so intense on a suborbital flight in real life, since the spacecraft is traveling slower than one coming down from an orbital would be.  Would it be a good idea to give a +5% or something to the reentry step of a Suborbital flight?

If so, perhaps the same should go for a mission where the rocket fails non-catastrophically and it never achieves orbit.

**Comment by hweimer** [comment]
Good point, but the game has a lot of such minor inconsistencies in order to keep the rules of the game simple enough.

Concerning your second point, the game currently branches launch aborts to recovery/landing and not to reentry. There was a discussion on this question in #330, after which we decided to leave it like that.

**Comment by peyre** [comment]
Fair enough.  There's something to be said for not complicating the game mechanics!
